src/components/Sidebar.js

```diff
@@ -33,7 +36,7 @@ function Sidebar() {
             </NavLink>
           </div>
           <div className="text-center row mt-auto pb-5">
-            <NavLink to="/" className="nav-link text-dark">
+            <NavLink to="" type="button" onClick={() => dispatch(logOut())} className="nav-link text-dark">
```

Comment:
`logOut` is a function that gets the dispatch method as an argument. Please, change this to
```javascript
onClick={() => dispatch(logOut)}
```
that is, without calling logOut. As it is now, it works, but it gives an error in the console.

Reply:
Change has been implemented. Thanks for the feedback!